### ISSUE TITLE
Incompatibility with grpc-java 1.33.0

### ISSUE THREAD

**Issue by djaneluz** [yidongnan/grpc-spring-boot-starter#441]
My application uses grpc.version = 1.33.0 and the server works just fine. But when I add this client lib and try to use it, I get the error:

```
***************************
APPLICATION FAILED TO START
***************************

Description:

An attempt was made to call a method that does not exist. The attempt was made from the following location:

    net.devh.boot.grpc.client.channelfactory.ShadedNettyChannelFactory.newChannelBuilder(ShadedNettyChannelFactory.java:73)

The following method did not exist:

    'io.grpc.internal.AbstractManagedChannelImplBuilder io.grpc.netty.shaded.io.grpc.netty.NettyChannelBuilder.defaultLoadBalancingPolicy(java.lang.String)'

The method's class, io.grpc.netty.shaded.io.grpc.netty.NettyChannelBuilder, is available from the following locations:

    jar:file:/C:/Users/d_luz/.m2/repository/io/grpc/grpc-netty-shaded/1.33.0/grpc-netty-shaded-1.33.0.jar!/io/grpc/netty/shaded/io/grpc/netty/NettyChannelBuilder.class

The class hierarchy was loaded from the following locations:

    io.grpc.netty.shaded.io.grpc.netty.NettyChannelBuilder: file:/C:/Users/d_luz/.m2/repository/io/grpc/grpc-netty-shaded/1.33.0/grpc-netty-shaded-1.33.0.jar
    io.grpc.ForwardingChannelBuilder: file:/C:/Users/d_luz/.m2/repository/io/grpc/grpc-api/1.33.0/grpc-api-1.33.0.jar
    io.grpc.ManagedChannelBuilder: file:/C:/Users/d_luz/.m2/repository/io/grpc/grpc-api/1.33.0/grpc-api-1.33.0.jar


Action:

Correct the classpath of your application so that it contains a single, compatible version of io.grpc.netty.shaded.io.grpc.netty.NettyChannelBuilder


Process finished with exit code 1

```
I'm using the last version 

        <dependency>
            <groupId>net.devh</groupId>
            <artifactId>grpc-client-spring-boot-starter</artifactId>
            <version>2.10.1.RELEASE</version>
        </dependency>

Which I noticed use an older version of io.grpc. I'm not sure if this is causing the problem.

Any ideas on how to fix this?

**Comment by ST-DDT** [comment]
This looks like an incompatibility due to the grpc-java update.
Your options:
1) Downgrade grpc-java
2) Wait for the next release of this library
3) Temporarily overwrite/recompile the offending class (ShadedNettyChannelFactory) by copying it to your code.

**Comment by ST-DDT** [comment]
See also:
https://github.com/grpc/grpc-java/issues/7552
and
https://github.com/grpc/grpc-java/pull/7564

**Comment by ST-DDT** [comment]
Waiting for grpc-java v1.33.1 release

**Comment by ST-DDT** [comment]
Fixed upstream - Update to [grpc-java v1.33.1](https://github.com/grpc/grpc-java/releases/tag/v1.33.1)
Could you please verify whether this fixes the errors for you?

**Comment by djaneluz** [comment]
Hello,

I tested and I no longer get the error message.

Thanks

Em sex., 30 de out. de 2020 às 10:17, ST-DDT <notifications@github.com>
escreveu:
